### DISCUSSION TITLE
model version for west coast of North America

### DISCUSSION THREAD

**Discussion by wimvandam** [bmvandoren/Nighthawk#13] [Q&A]
I'm wondering if anyone has a worked on a version for the west coast of the North America. The easiest first step would be to adjust the prior probabilities, the second step would be to add west coast species. Thanks for all your work on this project. 

Wim van Dam
Solvang, CA

**Comment by bmvandoren** [comment]
Hi Wim, thanks for getting in touch. This would certainly be of interest. Tuning the output by geography is definitely something that we would like to do. The easiest way to do this would be to provide the model with a filter list of species to include (or exclude) from consideration. In addition, we would want to add more West Coast species, as you say. We have some data from some West Coast species, but not nearly as much as from the east. Is this something you'd be interested to collaborate on?
